runtime/src/lib.rs

```diff
@@ -414,31 +447,21 @@ impl pallet_im_online::Trait for Runtime {
     type UnsignedPriority = ImOnlineUnsignedPriority;
 }
 
-parameter_types! {
-    pub OffencesWeightSoftLimit: Weight = Perbill::from_percent(60) * MaximumBlockWeight::get();
-}
-
-impl pallet_offences::Trait for Runtime {
+impl pallet_offences::Config for Runtime {
     type Event = Event;
     type IdentificationTuple = pallet_session::historical::IdentificationTuple<Self>;
     type OnOffenceHandler = Staking;
-    type WeightSoftLimit = OffencesWeightSoftLimit;
+    type WeightSoftLimit = ();
```

Comment:
I didn't find it in node-template. I restored it in case we introduced it by ourselves and it's needed.
Restored https://github.com/Joystream/joystream/pull/2558/commits/831899e778a890a59273fe155425f83f83ec6c23